content/08.transmission.md

```diff
@@ -218,62 +218,42 @@ A large analysis of over 11,000 COVID-19 patients hospitalized in 92 hospitals a
+Many governments issued stay-at-home orders, especially in the initial months of the crisis.
+However, data clearly indicates that these orders impacted different socioeconomic groups differently.
+In U.S. counties with and without stay-at-home orders, smartphone tracking indicated a significant decrease in the general population's mobility in April relative to February through March of 2020 (-52.3% and -60.8%, respectively) [@doi:10.1101/2020.05.03.20084624].
+A linear relationship was observed between counties' reduction in mobility and their wealth and health, as measured by access to health care, food security, income, space, and other factors [@doi:10.1101/2020.05.03.20084624].
+These counties were found to have much lower child poverty and household crowding and to be more racially segregated, and to have fewer youth and more elderly residents [@doi:10.1101/2020.05.03.20084624].
+Similar associations between wealth and decreased mobility were observed in cellphone GPS data from Colombia, Indonesia, and Mexico collected between January and May 2020 [@arXiv:2006.15195].
+These disparities in mobility are likely to be related to the role that essential workers have played during the pandemic.
+Essential workers are disproportionately likely to be female, people of color, immigrants, and to have an income below 200% of the poverty line [@url:https://mronline.org/wp-content/uploads/2020/06/2020-04-Frontline-Workers.pdf]. 
+Black Americans in particular are over-represented among front-line workers and in professions where social distancing is infeasible [@doi:10.1002/ajim.23145].
+Health care work in particular presents an increased risk of exposure to SARS-CoV-2 [@doi:10.1002/ajim.23145; @doi:10.1371/journal.pone.0232452; @url:https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020; @url:https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11; @url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf].
+In the United Kingdom, (South) Asians are more likely than their white cohorts to be medical professionals [@url:https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/], although BAME medical professionals are still disproportionately represented in the proportion of National Health Service staff deaths [@doi:https://www.hsj.co.uk/exclusive-deaths-of-nhs-staff-from-covid-19-analysed/7027471.article].
+Similar trends have been reported for nurses of color in the United States [@url:https://act.nationalnursesunited.org/page/-/files/graphics/0920_Covid19_SinsOfOmission_Data_Report.pdf].
```

Comment:
should this be nurses in general?

Reply:
The report says: "Just under a quarter (24.1 percent) of registered nurses in the United States are persons of color.52But as of Sept. 16, 2020, 124 (58.2 percent) of the 213 registered nurses who have died of Covid-19 and related complications are nurses of color, reflecting the broader disproportionate impact of Covid-19 on communities of color in the United States"
Does this work?
```suggestion
Similar trends have been reported for nurses, especially nurses of color, in the United States [@url:https://act.nationalnursesunited.org/page/-/files/graphics/0920_Covid19_SinsOfOmission_Data_Report.pdf].
```